docs/src/assets/scss/components/rules.scss

```diff
@@ -65,7 +65,7 @@
     }
 }
 
-.rule {
+.rule:not(.token) {
```

Comment:
Doesn't this change affect all places with rule class? 👀  Have we tested at other scenarios as well? 

Is something similar happening at https://eslint.org/docs/head/rules/function-paren-newline?

Reply:
> Doesn't this change affect all places with rule class? 👀 Have we tested at other scenarios as well?

  we have explicitly used `rule` class name in [rule.macro.html](https://github.com/eslint/eslint/blob/b6ab030897d2e8b314b33a6502346a4ac45bb8da/docs/src/_includes/components/rule.macro.html#L4) file only. So it will not affect I guess. 



> not similar to https://eslint.org/docs/head/rules/function-paren-newline. 

It is already mentioned in the PR description `it's not the same problem as broken line number.`